Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/PropertyCRCCtrl.h

```diff
@@ -56,24 +57,41 @@ namespace AzToolsFramework
         
     };
 
-    // note:  QT Objects cannot themselves be templates, else I would templatize creategui as well.
-    class U32CRCHandler  
+    template<class ValueType>
+    class CRC32HandlerCommon : public PropertyHandler<ValueType, PropertyCRCCtrl>
+    {
+    public:
+        AZ::u32 GetHandlerName(void) const override { return AZ::Edit::UIHandlers::Crc; }
+        bool IsDefaultHandler() const override { return true; }
```

Comment:
I don't think we should register this as a default handler because then any `AZ::u32` field will use this instead of the normal integer property handler. It looks like the previous behavior relied on the user explicitly requesting this handler. But I do like the improvement of making the property handler for `AZ::Crc32` being a default handler.